view/template/acquisition/yt2.php

```diff
@@ -66,20 +45,32 @@
 </section>
 <section class="claim section">
```

Comment:
(This isn't the correct line number, but the reward values still need to be plugged in dynamically.)